src/packages/media/media/components/input-image-cropper/image-cropper-focus-setter.element.ts

```diff
@@ -9,36 +12,48 @@ export class UmbImageCropperFocusSetterElement extends LitElement {
 	imageElement!: HTMLImageElement;
 
 	@query('#wrapper')
-	wrapperElement?: HTMLImageElement;
+	wrapperElement?: HTMLElement;
 
 	@query('#focal-point')
-	focalPointElement!: HTMLImageElement;
+	focalPointElement!: HTMLElement;
+
+	@state() private isDraggingGridHandle = false;
+
+	@state() private coords = { x: 0, y: 0 };
 
 	@property({ attribute: false })
-	focalPoint: UmbImageCropperFocalPoint = { left: 0.5, top: 0.5 };
+	get focalPoint() {
+		return this.#focalPoint;
+	}
+	set focalPoint(value) {
+		this.#focalPoint = value;
+		this.#onFocalPointUpdated();
+	}
```

Comment:
These need to be moved around according to Lit 3 warnings:

```suggestion
	set focalPoint(value) {
		this.#focalPoint = value;
		this.#onFocalPointUpdated();
	}
	get focalPoint() {
		return this.#focalPoint;
	}
```